File: 프로그래머스/Level 0/옹알이1-ex.java
class Solution {
    public int solution(String[] babblings) {
        int answer = 0;
        String[] words = {"aya", "ye", "woo", "ma"};
        
        for (String babbling : babblings) {
            for (String word : words) {
                if (babbling.contains(word)) 
                    babbling = babbling.replace(word, "0");
            }
            
            babbling = babbling.replace("0", "");
            
            if (babbling.isEmpty()) answer++;
        }
        
        return answer;
    }
}

File: 프로그래머스/Level 0/옹알이1-ex.py
def solution(babblings):
    count = 0
    words = ["aya", "ye", "woo", "ma"]
    for babbling in babblings:
        for word in words:
            if word * 2 not in babbling:
                babbling = babbling.replace(word, ' ')
                
        if len(babbling.strip()) == 0:
            count += 1
    
    return count